code/VersionedDataObject.php

```diff
@@ -80,39 +74,42 @@ public function isPublished()
 
         return (bool) DB::query("SELECT \"ID\" FROM \"{$table}_Live\" WHERE \"ID\" = {$this->owner->ID}")->value();
     }
+
     /**
-     * @param $value
-     * @return string
-     */
-    protected function getBooleanNice($value)
-    {
-        return $value ? 'Yes' : 'No';
-    }
-    /**
-     * @return mixed
-     */
-    public function isPublishedNice()
-    {
-        return $this->getBooleanNice($this->isPublished());
-    }
-    /**
-     * @return mixed
+     * @return HTMLText
      */
-    public function isModifiedNice()
+    public function getCMSPublishedState()
     {
-        return $this->getBooleanNice($this->stagesDiffer('Stage', 'Live'));
+        $html = new HTMLText('PublishedState');
```

Comment:
I'm not too happy with doing this casting manually, but after digging through SilverStripe's source, there doesn't appear to be a way to using automatic casting for summary fields at the moment.